sdk/servicebus/service-bus/test/stress/test-resources.json

```diff
@@ -1,77 +0,0 @@
-{
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
why don't we need this?

Reply:
Much earlier, the deployment scripts didn't auto-build the bicep file, so we had to check in the ARM template generated from the bicep file. Now the automation just treats the bicep file as the source of truth and generates the ARM template on the fly at deployment time.